src/pandroid/app/src/main/java/com/panda3ds/pandroid/view/controller/ControllerNode.java

```diff
@@ -0,0 +1,22 @@
+package com.panda3ds.pandroid.view.controller;
+
+import android.view.View;
+import androidx.annotation.NonNull;
+import com.panda3ds.pandroid.math.Vector2;
+
+public interface ControllerNode {
+	@NonNull
+	default Vector2 getPosition() {
+		View me = (View) this;
```

Comment:
Zhould probably be called view instead of me

Reply:
```suggestion
		View view = (View) this;
```